Cargo.toml

```diff
@@ -60,7 +60,9 @@ tokio = {version = "1", features = ["full"]}
 tokio-util = { version = "0.7.4", features = ["compat"]}
 
 [features]
-default = ["signature-pgp", "async-futures", "with-file-async-tokio"]
+default = ["signature-pgp", "async-futures", "with-file-async-tokio", "compress"]
+gzip = ["libflate"]
+compress = ["gzip", "zstd"]
```

Comment:
I do think it makes sense to split out these features, but I'm curious about your motivation, is it primarily just about code size / compilation time?  Or system dependencies that complicate cross-compilation?

I'm not sure about the "compress" feature as AFAIK nearly all RPMs use some form of compression, disabling it entirely is not very helpful.

Reply:
Well, I am writing `antivirus` and I need `rpm` package parser. In my project I use other library for ungziping.

Reply:
It sounds pretty fair to have 2 staged features, with all compression algorithms and individuals selectable

Reply:
Removed